modules/node_modules/@ncigdc/containers/SmartSearchPage.js

```diff
@@ -94,6 +94,20 @@ class SmartSearchComponent extends React.Component {
           this.Error = null;
 
           this.setQuery();
+
+          document.querySelector('.btn-search-query').onclick = () => {
```

Comment:
Is that a problem when using level 0 events? I've never thought of it when using them.